File: src/main/java/org/humeurvagabonde/playtime/PlaytimeApplication.java
package org.humeurvagabonde.playtime;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.http.server.reactive.HttpHandler;
import org.springframework.web.reactive.function.server.RouterFunctions;

@SpringBootApplication
public class PlaytimeApplication {

    public static void main(String[] args) {
        SpringApplication.run(PlaytimeApplication.class, args);
/*
        HttpHandler httpHandler = RouterFunctions.toHttpHandler(route);
        ReactorHttpHandlerAdapter adapter = new ReactorHttpHandlerAdapter(httpHandler);
        HttpServer server = HttpServer.create(HOST, PORT);
        server.newHandler(adapter).block();
*/
    }

}

File: src/main/kotlin/org/humeurvagabonde/playtime/PlaytimeApplication.kt
package org.humeurvagabonde.playtime

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
class PlaytimeApplication

fun main(args: Array<String>) {
    SpringApplication.run(PlaytimeApplication::class.java, *args)
}
